pkg/ebpf/events_pipeline.go

```diff
@@ -243,8 +243,10 @@ func (t *Tracee) processEvents(ctx gocontext.Context, in <-chan *trace.Event) (<
 				// Although kernel filters shouldn't submit such events, we do this check to be on the safe side.
 				// For example, it might be that a new cgroup was created, and not by a container runtime,
 				// while we still didn't processed the cgroup_mkdir event and removed the cgroupid from the bpf container map.
-				// Note: this check should be placed after processEvent() so cgroup_mkdir event is processed
-				continue
+				id := events.ID(event.EventID)
+				if id != events.CgroupMkdir && id != events.CgroupRmdir {
```

Comment:
Can you add a comment describing that this is derived into container_create and container_removed respectively? Wasn't obvious at a glance.